packages/runner/src/tasks/data-factory-task.ts

```diff
@@ -33,7 +33,7 @@ export class DataFactoryTask {
 
             return { results: scanResults };
         } else {
-            return { results: [], error: axeScanResults.error };
+            return { results: [], error: axeScanResults.error, unscannable: axeScanResults.unscannable === true ? true : false };
```

Comment:
Is it ok to just use unscannable: axeScanResults.unscannable, whats harm in having scannable as  'undefined'?

Reply:
It is OK I think.